plugins/modules/create_kafka.py

```diff
@@ -54,14 +54,7 @@
         description: Billing model for the Kafka instance.
         required: true 
         type: str
-    instance_type:
```

Comment:
why remove the instance type?

Reply:
To long for comment on github. We need to do some small AMS training in the team.